### PR TITLE
This PR contains the reviewed code of perfmetrics

### PR DESCRIPTION
**Pull request by etiksha**

### CODE REVIEW

**Review by vadlakondaswetha — APPROVED** [review]
Code is already reviewed in google3. Approving them

**Review by vadlakondaswetha — APPROVED** [review]

**Review by avidullu — APPROVED** [review]
LGTM. Will merge after pull/684 is merged.

**Pull request merged**
